uint8 Robot
uint8 Obj
---
bool status